installer/pyinstaller/build-linux.sh

```diff
@@ -11,6 +12,13 @@ if [ "$python_version" = "" ]; then
     python_version="3.7.9";
 fi
 
+if ! [ "$nightly_build" = "" ]; then
+    echo "Building native installer with nightly build"
+    is_nightly="true"; else
```

Comment:
nit: is this how `else` written in bash files? If we can move this into new line it will be better.

Reply:
Thanks! Fixed this.